### DISCUSSION TITLE
Ability for users to manually unlock achievements (in exchange of no points)

### DISCUSSION THREAD

**Discussion by TheMrGmk** [RetroAchievements/RAWeb#1516] [Ideas]
Hello there,
I was having a conversation about this topic on the discord forum, and someone suggested to try to get in contact with you devs ^^
Basically, I love the idea of having retro achievements for retrogames, but I was thinking about the possibility to be able to use my old save files and retroactively get achievements that I already "unlocked", to not ditch hours of progress already made in games.
I stumbled on some really old threads and it has come to my understanding that in the past this was possible, but to prevent people from cheating on the leaderboards this feature is no longer available (which makes sense since the integrity of the leadeboards is fundamental).

As some people have pointed out, disabling save protection for "unranked" users that are not registered on the leaderboards while ideal would be impractical as it would probably require an unrealistic amount of work dev side and would be open to vulnerabilities.
But maybe having the ability to manually unlock achievements that I'm certain I've obtained in the past, for no amount of points, and continue to unlock the subsequent ones on trigger events like normal would be more feasible.

So my question is: would you be open to add the possibility to manually unlock achievements, which don't give any points ? Maybe by also excluding them from the achievement section or by granting this possibility only to "unranked" users. In this way leaderboards would remain clean but whoever has a lot of games with tens of hours of progress could experience the extra challenge offered by achievements while not ditching that progress.
There are people (like myself) that see achievements as a way to check on personal progress/have a reference sheet for possible future challenges mainly for personal enjoyment, and are not really interested in competition, so I can see the willingness of some people to have a more personal experience in exchange of the more competitive aspects of achievements.

Sorry if my post is convoluted but english is not my native language, hope I didn't sound harsh : )

**Comment by jackgllghr** [comment]
I would also like to see this, I just completed Bubble Bobble on my Miyoo Mini and have missed multiple achievements along the way(should get achievements for each set of levels, and I just beat level 100 and rolled credits). 

It's just a bit silly that because I was offline/network issue when waking the device, that I have achievements for completing level 70 but not 60 and I can't just go to the website and unlock them for myself. 

I would just flag them as manually unlocked, and if some queries need to be adjusted for the leaderboards and point tracking then we could adjust them. I'd be happy to code it if the team are onboard with the change.

**Reply by wescopeland** [reply]
Having a way for users to self-manual unlock and have achievements sitting in this status adds a lot of downstream complexity all over the site.

I think I'd rather see us streamline the manual unlock process, with it possibly living as an on-site feature rather than users handling it on Discord or DMing the RAdmin account.

**Reply by jackgllghr** [reply]
Thanks for the quick reply! I'd imagine with the rise of handheld PCs and retro handhelds that's there's been a rise in manual unlock requests of recent? It seems like a very unscalable solution if mods need to intervene for every request. 

For my specific issue, I'd imagine it could be alleviated by having linked achievements so if level 5 achievement is unlocked that it should retroactively unlock level 1,2,3,4 for example, so maybe that's a more suitable solution(which I'd imagine would require a lot of work on the various achievement developers part). 

Alternatively, offline sync would be cool to have but I don't know the history of why they need to be unlocked online so I imagine there's additional complexity there is there?
